engine/src/main/java/org/terasology/rendering/dag/tasks/LookThroughDefaultCameraTask.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.dag.tasks;
+
+import org.terasology.rendering.dag.RenderPipelineTask;
+
+import static org.lwjgl.opengl.GL11.GL_MODELVIEW;
+import static org.lwjgl.opengl.GL11.GL_PROJECTION;
+import static org.lwjgl.opengl.GL11.glLoadIdentity;
+import static org.lwjgl.opengl.GL11.glMatrixMode;
+
+/**
+ * Instances of this class reset the ModelView and Projection matrices to identity matrices,
+ * as per opengl default.
+ *
+ * WARNING: RenderPipelineTasks are not meant for direct instantiation and manipulation.
```

Comment:
If these classes shouldn't be used directly should there be some clever way to prohibit a user from acquiring an instance? Like a package private factory method, more access limited `execute`, or something otherwise sneaky.